### PR TITLE
[Fixes #2] Publish container to ghcr.io

### LINKED ISSUE #2: Rename fork to `packer-plugin-cross`
## Purpose
To distinguish ourselves from the original and as we aim to extend support to more than just ARM + to be compatible with plugins' ecosystem, we should rename.

## Solution
* Rename the repository
* Rename plugin details in the code

### PR DESCRIPTION
**Pull request by dbast**
## Purpose
Fixes publication of Docker images into GHCR.

## Scope
* CI workflow about Docker image publication.
* Also remove Arch Linux armv7/v8 docker containers.

## Note
PR is against the chore/2/rename-the-project branch.

### CODE REVIEW

**Comment by michalfita** [comment]
@dbast thanks for addressing this, just need to make sure it'll publish under Packages in this project.

Why removing `boards/armv7/alpine.json`? That one is not linked with Docker post-production?

**Comment by dbast** [comment]
right, just discovered `boards/armv7/alpine.json` when removing the armv7/8 docker images... removed it due to being outdated, not being a real `board` and not part of any CI test flow. Could also be done via separate PR.

**Comment by dbast** [comment]
The docker container can later be also be minimised once the plugin binaries are published.

**Comment by dbast** [comment]
PR requires also some README.md updates before/after merge.

**Comment by dbast** [comment]
updated to keep boards/armv7/alpine.json

File: builder/artifact.go
Comment (michalfita) [inline builder/artifact.go:1]: I've sneaked this change I didn't manage to push before you merged #4.

**Review by michalfita — APPROVED** [review]
LGTM now 👍🏼

**Comment by dbast** [comment]
now with README update.

**Comment by michalfita** [comment]
I have no idea why `vagrant` workflow fails. Finding VM in `gurumeditation` state indicates issue with booting of the VM. Should we disable it for now?

**Comment by dbast** [comment]
vagrant issues are know... https://github.com/orgs/community/discussions/73013 it currently sometimes work and sometimes not... seems like not all action runners have the same VM image version ... the vagrant failure in this PR can be ignored.

**Comment by dbast** [comment]
merging to see if ghcr.io token setup etc work

**Comment by dbast** [comment]
got this published in my fork from the development branch without further relevant changes https://github.com/dbast/packer-plugin-cross/pkgs/container/packer-plugin-cross

**Comment by dbast** [comment]
triggering another build https://github.com/michalfita/packer-plugin-cross/actions/runs/7332267593

**Comment by dbast** [comment]
another option would be adding this to the workflow (though not required in my repo fork):
```
permissions:
  packages: write
```

**Comment by michalfita** [comment]
Makes sense to add this.

**Pull request merged**